_plugins/medium_post_display.rb

```diff
@@ -0,0 +1,23 @@
+require 'feedjira'
+module Jekyll
+  class MediumPostDisplay < Generator
+    safe true
+    priority :high
+def generate(site)
```

Comment:
Since you are introducing some new logic in Ruby which wasn't there before, we would need a test file for it. I'm not sure if you know how to write tests in Ruby so its okay if you want to write the tests in another PR after this one is merged, but we will need some tests.

This would also mean enhancing our CI setup a bit to run ruby tests so let me know if you want to work on those too.